### PR TITLE
Add optional thresholds support to Namesake CLI #1

### PR DESCRIPTION
**Pull request by mlanicaputo**
Addresses #1

This PR adds optional thresholds to the Namesake CLI. This enables users to use custom similarity thresholds when evaluating identifier similarity, in addition to supporting the previous default values.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,12 +50,18 @@ pip install -r /namesake/requirements.txt
...
 
 ## 🚀 Running Namesake:
-To run Namesake on the file test1.py
+To run Namesake on the file test1.py with default similarity thresholds:
 
 ```sh
 python namesake.py test1.py
 ```
 
+To run Namesake on the file test1.py with custom similarity thresholds `(0.5, 0.6 0.7)`:
```
Comment (nalmadi) [inline README.md]: Did you mean (0.5, 0.6, 0.7)?

File: namesake.py [hunk 2]
```diff
@@ -275,14 +275,32 @@ def print_semantic_warnings(semantic_similarity, identifiers_lines, threshold):
...
+            # verify that the provided thresholds are between 0 and 1
+            for threshold in (orth_threshold, phon_threshold, sem_threshold):
+                if not (threshold > 0 and threshold < 1):
+                    print("Aborted: Optional thresholds must be between 0 and 1")
+                    sys.exit(1)
 
     # open a file passed by the command line
     file = open(sys.argv[1])
     code = file.read()
+    file.close()
```
Comment (nalmadi) [inline namesake.py:303]: Good catch!

File: README.md (hunk 1 above)
Comment (nalmadi) [reply README.md]: Also, can we add a description of the order? like: [orth_threshold] [phon_threshold] [sem_threshold]

**Review by nalmadi — CHANGES_REQUESTED** [review]
Great work checking for validity and updating readme to reflect changes.  I will approve this as soon as you make the suggested changes to the readme file.  Thanks @mlanicaputo!

File: README.md (hunk 1 above)
Comment (mlanicaputo) [reply README.md]: Good call; I can include this in the usage description instead of having two different examples.

**Comment by nalmadi** [comment]
Thanks @mlanicaputo!

**Pull request merged**